src/python/pants/backend/python/tasks2/pytest_run.py

```diff
@@ -77,15 +78,20 @@ def failed_targets(self):
     return self._failed_targets
 
 
-class PytestRun(TestRunnerTaskMixin, Task):
+class PytestRun(TestRunnerTaskMixin, PythonExecutionTaskBase):
 
   @classmethod
   def register_options(cls, register):
     super(PytestRun, cls).register_options(register)
-    register('--fast', type=bool, default=True,
```

Comment:
This flag is in a release, so we have to deprecate it, we can't just eliminate it.